### ISSUE TITLE
[Bug] Custom Center Pose

### ISSUE THREAD

**Issue by SoulMaril** [opentrack/opentrack#594]
Hey,

When you set a Custom Center Pose it does not have any effect on changing the center pose  instead it displaces the red cross of calibration.

**Comment by Len62** [comment]
Since the red cross represents the center, wouldn't it be "custom" if it is offset ?

**Comment by sthalik** [comment]
Red cross isn't meant to move since it's emitted by the tracker. Center pose only takes place after all of tracker's processing takes place.

**Comment by SoulMaril** [comment]
I just want to offset my view in the game. I was using this feature in WarThunder, IL2 Sturmovik to offset my gunsight  a little bit up on the Y axis. Now it is not working anymore. As far as I withnessed when I enter some offset value in that box, say 2 cm, I found out that the Red dot is moving downwards. Try yourself.

If there is another way to offset the view in game tell me. I will appreciate.
